install.sh: Attempt to restore SELinux attributes

drinkcat opened the pull request:
Took me a while to figure out... Might be helpful to others.

In particular, this is required on Rocky Linux 9.5 for docuum to run as a systemd service.

**Status:** Ready

**Fixes:** N/A

stepchowfun commented:
Thanks for your PR! I'm wondering if we should put that command in the installation script so the user doesn't have to run it. We'd have to do it in a way that no-ops on non-applicable platforms.

drinkcat committed:
install.sh: Attempt to restore SELinux attributes

In particular, this is required on Rocky Linux 9.5 for docuum to
run as a systemd service.

drinkcat commented:
> Thanks for your PR! I'm wondering if we should put that command in the installation script so the user doesn't have to run it. We'd have to do it in a way that no-ops on non-applicable platforms.

Should work (I didn't test on a machine without `restorecon`).

I didn't make the error an install failure, it might not be a problem depending on configuration...

stepchowfun commented:
Looks good to me! The CI jobs will test it in environments that don't have SELinux/`restorecon`.

Thank you for contributing to Docuum!
